File: alanvianadm/ruby/fibonacci.rb
def fibonacci(number)
    a = 0
    b = 1
    number.times do
	temp = a
	a = b
	b = temp + b
    end
    return a
end

puts "Digite o tamanho da sequencia de fibonacci desejada: "
seq = gets.to_i

seq.times do |number|
    result = fibonacci(number)
    puts result
end

File: alanvianadm/python/fibonacci.py
initialNumber = int(raw_input("Digite o primeiro numero:  "))
lastNumber = int(raw_input("Digite o ultimo numero "))

def fibonacci(n):
    a = 0
    b = 1
    for i in range(0, n):
	temp = a
	a = b
	b = temp + b
    return a

for c in range(initialNumber, lastNumber):
    print(fibonacci(c))
